.github/workflows/e2e.yaml

```diff
@@ -42,8 +42,7 @@ jobs:
           fi
       - name: Run install terraform
         run: |
-          mkdir -p ./examples/install/.terraform/plugins/registry.terraform.io/fluxcd/flux/0.0.1/linux_amd64
-          cp ./bin/flux ./examples/install/.terraform/plugins/registry.terraform.io/fluxcd/flux/0.0.1/linux_amd64/terraform-provider-flux_v0.0.1
+          make install
           cd examples/install
```

Comment:
Suggestion to the end-to-end test:
Add a separate test where you install the latest stable and then run it again with the newly built provider. Should maybe not be a required check, but good to know if something breaks in upgrades.

Reply:
Yeap I will try to add this later in its own PR.